tests/run-tests.sh

```diff
@@ -15,6 +15,6 @@ ln -snf "test.wake.template" "$api_scala_sifive_path/tests/test.wake"
 
 # This is gross because we don't have a way of telling Wit about Ivy
 # dependencies for testing-only packages.
-./scala/coursier fetch --cache ./ivycache org.json4s::json4s-native:3.6.7
+./scala/coursier fetch --cache ./ivycache --scala-version 2.12.8 org.json4s::json4s-native:3.6.7
```

Comment:
Just note that the Scala version transforms `org.json4s::json4s-native:3.6.7` into `org.json4s:json4s-native_2.12:3.6.7`, you can just specify it that way if you want.

This is documented here: https://github.com/sifive/api-scala-sifive#dependencies

Reply:
Not saying this should change, just letting you know!

Reply:
Ah, I see what I was doing wrong; I had tried something similar but I left the double colon in, but now I understand that the difference between the double and single colon is whether the Scala version is appended.

I'm hoping this all goes away once we have a way of supporting multiple, arbitrary `ivydendencies.json` files in the some Git repo (possibly as separate projects).